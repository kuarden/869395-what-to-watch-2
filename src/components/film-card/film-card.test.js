import React from 'react';
import renderer from 'react-test-renderer';
import FilmCard from './film-card.jsx';

it(`FilmCard renders correctly`, () => {
  const tree = renderer.create(<FilmCard
    film={{
      id: 1,
      name: `Seven Years in Tibet`,
      genre: `Adventure`,
      previewImage: `https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg`,
      videoLink: `http://peach.themazzone.com/durian/movies/sintel-1024-surround.mp4`}}
  />).toJSON();

  expect(tree).toMatchSnapshot();
});
