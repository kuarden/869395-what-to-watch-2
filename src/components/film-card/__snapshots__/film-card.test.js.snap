// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmCard renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <div
    className="small-movie-card__image"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <video
      height="175"
      poster="https://htmlacademy-react-2.appspot.com/wtw/static/film/preview/seven-years-in-tibet.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
    onClick={[Function]}
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Seven Years in Tibet
    </a>
  </h3>
</article>
`;
